### PR TITLE
fix conflict

### PR DESCRIPTION
**Pull request by wenbinc-Bin**

### CODE REVIEW

**Comment by uartie** [comment]
Build failure caused by bad submodule commit.  Please only include the remote submodule updates in commit without applied local patches.

**Comment by xhaihao** [comment]
Maybe there is a memory leak now since commit 3bbe0c210b05fc6fbd7b1d4bbd8479db7f2cf957.  In the past, ExtParam is not used in encoding path.

**Comment by xhaihao** [comment]
Please refer to commit 1cdbccaa163eb3bd50403ecc75fc3da9d5d75c02 for the leak fix for mfxEncodeCtrl::Payload

File: patches/0058-libavcodec-qsvenc.c-add-ROI-support-to-qsv-encoder.patch [hunk 1]
```diff
@@ -47,10 +47,10 @@ index 5efb5c538b5c..c4e40591e2c2 100644
...
              memset(&cur->enc_ctrl, 0, sizeof(cur->enc_ctrl));
-@@ -1363,6 +1370,12 @@ static int get_free_frame(QSVEncContext *q, QSVFrame **f)
-         av_freep(&frame);
+@@ -1360,6 +1367,12 @@ static int get_free_frame(QSVEncContext *q, QSVFrame **f)
          return AVERROR(ENOMEM);
      }
+     frame->enc_ctrl.Payload = frame->payloads;
 +    frame->enc_ctrl.ExtParam = av_mallocz(sizeof(mfxExtBuffer*) * QSV_MAX_ENC_EXTPARAM);
 +    if (!frame->enc_ctrl.ExtParam) {
 +        av_free(frame->enc_ctrl.Payload);
```
Comment (xhaihao) [inline patches/0058-libavcodec-qsvenc.c-add-ROI-support-to-qsv-encoder.patch:56]: Needn't call av_free() here.

**Comment by wenbinc-Bin** [comment]
I will resubmit PR
Thanks

**Comment by uartie** [comment]
Test failure is caused by intel/media-driver#1303

**Pull request merged**